credentials/src/test/java/me/uport/sdk/credentials/CredentialsTest.kt

```diff
@@ -83,6 +87,58 @@ class CredentialsTest {
         assertThat(jwt).isEqualTo(expectedJWT)
     }
 
+    @Test
+    fun `create basic verifiable credential`() = runBlocking {
+
+        val signer = KPSigner("74894f8853f90e6e3d6dfdd343eb0eb70cca06e552ed8af80adadcc573b35da3")
+        val did = "did:ethr:${signer.getAddress()}"
+
+        val cred = Credentials(did, signer, clock = object : ITimeProvider {
+            override fun nowMs() = 1485321133000L
```

Comment:
Is there a specific reasoning or benefit to overriding `nowMs()` instead of specifying `nbf`?  Is it to make sure the test is really testing only the required parameters?

Reply:
This is a way to provide a custom clock to the Credentials instance.

The general use-case is for testing but, with this pattern, one can also do `was valid at <timestamp>` kind of checks on JWTs.

By providing an `nbf` param directly, this would be testing a different thing.